src/pages/team.js

```diff
@@ -1,41 +1,134 @@
+      if (filterState === e.target.textContent) {
+        // The button was already selected.
+        setFilterState(null);
+      } else {
+        setFilterState(e.target.textContent);
+      }
+    }
+  };
+  let team;
+  if (filterState) {
+    team = teamDataNodes.filter((person) => person.Role === filterState);
+  } else {
+    team = teamDataNodes;
+  }
   return (
     <GeneralLayout>
-      <pre>{JSON.stringify(data, null, 4)}</pre>
+      <section className="team--content-section">
+        <div className="inner">
+          <div className="layout-div">
+            <div className="content">
+              <h2>You've never met a team like this one</h2>
+              <p className="body">
+                We are leaders in technology, design, and strategy for
+                government digital services. We combine our diverse expertise
+                and backgrounds to bring value to our customers and each other.
+              </p>
+            </div>
+            <div className="content">
+              <h3>Purpose driven. Uniquely talented. Remarkably human.</h3>
+              <p className="body">
+                People who work with us say there’s “something magical” about
+                our team. We are good listeners, strategic thinkers, honest
+                communicators, and problem solvers. (We’re also cheerful and
+                kind, which is a nice bonus.) We each care deeply about the
+                impact we have on those around us.{' '}
+              </p>
+              <p className="body">
+                We are the humans of CivicActions. It’s nice to meet you!
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section className="team--filter-btns-section">
+        <div className="inner">
+          <FilterButtons
+            parentCallback={callback}
+            team={teamDataNodes}
+            filterState={filterState}
+          />
+        </div>
+      </section>
+
+      <section className="team--staff-teasers-grid-section">
+        <div class="inner">
+          <div className="section__teaser-grid">
+            <TeamTeasers team={team} />
+          </div>
+        </div>
+      </section>
+
+      <StaffQuote
+        quote="My teammates are brilliant innovators, high performers, and conscientious human beings. Together we’re working to improve lives."
+        name="Alanna Casey"
+        role="Technical Writer"
+        img={alanaCaseyProfilePicture}
+      />
     </GeneralLayout>
   );
 };
 
-export const query = graphql`
-  {
-    allStrapiStaffProfile {
-      nodes {
-        id
-        Name
-        Pronunciation
-        Personal_Pronouns
-        Role
-        Path
-        Location
-        Quote
-        Social {
-          Title
-          Url
-        }
-        Specialty {
-          Specialty
-          id
-        }
-        Body
-      }
-    }
-  }
-`;
+const TeamTeasers = ({ team }) => {
```

Comment:
@alexiscott 
This looks great. Wondering if we should abstract TeamTeasers out into a separate component so we can use it to display other team lists. Eg. a list of team-members in a case study page.
This can be done in a separate ticket though if needed.

Reply:
Yes, you are right @iris-i . thanks. happy to do that once we get get to it. CC @jaube05 in case she comes across it first.